app/controllers/dashboard_controller.rb

```diff
@@ -1,12 +1,9 @@
 class DashboardController < ApplicationController
   def index
-    @user = current_user
-    # @user_costumes = @user.costumes
-    # @user_costume_ids = []
-    # @user_costumes.each do |costume|
-    #   @user_costume_ids << costume.id
-    # end
-
-    # @received_bookings = Booking.where(@user_costume_ids.include(costume_id))
+    @user = policy_scope(User)
+    @my_costumes_ids = @user.costumes.map do |costume|
+      costume.id
+    end
+    @received_bookings = Booking.where(costume_id: @my_costumes_ids)
```

Comment:
Could you please explain this syntax:
1) Is @user = policy_scope(User) the equivalent to @restaurants = policy_scope(Restaurant) in the lecture slides ? I guess this is also why we have a file UserPolicy, but not a file DashboardPolicy?
2) Does this work: ... .where(costume_id: @my_costumes_ids) ? Can you really pass an array of ids as argument?